readers-frontend/src/components/Article.astro

```diff
@@ -50,31 +54,54 @@ const {
             <>
               <a class="h-full w-1/2" href={articleUrl}>
                 <img
-                  class="aspect-video rounded-xl object-cover"
+                  class="aspect-video w-full rounded-xl object-cover"
                   src={imageUrl}
                   alt="Article Thumbnail"
+                  onerror={`if (this.src !== ${imagePlaceHolder}) this.src = ${imagePlaceHolder}`}
                 />
               </a>
-              <div class="border-color ml-3 h-full w-1/2 border-t-2">
-                <a
-                  class="text-l category-text category-color uppercase xl:text-xl 2xl:text-2xl"
-                  href={`/${categoryLink}`}
-                >
-                  {category}
-                </a>
+              <div
+                class={`border-color ml-3 h-full w-1/2 ${
+                  !isInMuseBanner && "border-t-2"
+                }`}
+              >
+                {!isInMuseBanner && (
+                  <a
+                    class="text-l category-text category-color uppercase xl:text-xl 2xl:text-2xl"
+                    href={`/${categoryLink}`}
+                  >
+                    {category}
+                  </a>
+                )}
                 <div class="mb-2">
                   <a href={articleUrl}>
-                    <p class="text-l xl:text-xl 2xl:text-2xl">{headline}</p>
+                    <p
+                      class={`text-l xl:text-xl 2xl:text-2xl ${
+                        isInMuseBanner && "text-white"
```

Comment:
this is fine but we could have an optional `textColor` input to the component that is defaulted to black. Then you won't have to add these checks here. You'd only have to initialise the variable as we do with `category-color`